lib/wickedbase.pm

```diff
@@ -30,29 +31,37 @@ sub assert_wicked_state {
 
 sub get_ip {
```

Comment:
since you touch this function anyway , maybe you can implement switch to a flag "mask" instead of "no_mask" ? ( Remember we discussed this ? )

Reply:
I remember and will do it.